person/src/com/person/core/containers/PersonList.java

```diff
@@ -0,0 +1,46 @@
+package com.person.core.containers;
+
+import com.person.core.model.BasePerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonList {
+
+    private List<BasePerson> people = new ArrayList<>();
+
+    /**
+     * Adds person to the list.
+     * @param person
+     */
+    public void add(BasePerson person)
+    {
+        people.add(person);
+    }
+
+    /**
+     * Get person by name and surname.
+     * @param name
+     * @param surname
+     * @return
+     */
+    public BasePerson get(String name, String surname)
+    {
+        for(BasePerson i: people)
```

Comment:
@tatevikb usually in foreach is better to write : for(BasePerson person: persons)

Reply:
Ok!